.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: nim
+sudo: required
```

Comment:
I hate to use `sudo` in TravisCI if I can avoid it, because it means we have to wait a lot longer to acquire a build-node. Without `sudo`, we get to use Travis VMs.

Well, I guess TravisCI+sudo w/ Docker could work well. I haven't tried that.